src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Search/NewAzureOperationalInsightsComputerGroupCommand.cs

```diff
@@ -73,11 +73,6 @@ protected override void ProcessRecord()
                 Tags = new List<Tag>() { new Tag() { Name = "Group", Value = "Computer" } }
             };
 
-            if (!SearchCommandHelper.IsListOfComputers(this.Query))
```

Comment:
IMHO, I don't think extra test coverage is necessary in this change. What does the cmdlet do is to save the computer group Query to a table in OperationalInsights. The cmdlet doesn't care the format inside the query as long as it is saved to the table succesfully, which existing test cases in azure-powershell already covered.

There are test coverage on OperationalInsights side to make sure both the new and old query syntaxes work as expected. So the query functionalities are also covered.